rubicon/sklearn/loggers.py

```diff
@@ -1,33 +1,22 @@
-from .base_logger import BaseEstimatorLogger
+from rubicon.exceptions import RubiconException
+from rubicon.sklearn.base_logger import BaseEstimatorLogger
 
 
-class StandardScalerLogger(BaseEstimatorLogger):
-    """
-    This is an example, remove before final PR.
-    """
+class FilteredLogger(BaseEstimatorLogger):
```

Comment:
now that we know we're not going to be including a lot of custom loggers, it might make more sense to rename this file to `filter_logger`